backendservice/src/main/java/com/google/gpay/virtualqueue/backendservice/repository/VirtualQueueRepository.java

```diff
@@ -34,6 +34,8 @@
 
     public List<Shop> getAllShops();
 
+    public Long getWaitingTime();
```

Comment:
Nit. It's recommened to use primitives as opposed to boxed data types.
use long instead of Long ?

Reply:
Absolutely. Used long now.